kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -50,6 +54,12 @@ public void initChannel(SocketChannel ch) {
         LOGGER.trace("Connection from {} to my address {}", ch.remoteAddress(), ch.localAddress());
 
         ChannelPipeline pipeline = ch.pipeline();
+
+        if (sslContext != null) {
+            LOGGER.debug("Adding SSL handler");
+            pipeline.addLast(sslContext.newHandler(ch.alloc()));
+        }
```

Comment:
Again my dislike of `null` checks showing is there nicer way using `Optional`? 

Something like `sslContext.ifPresent(() -> {}
```suggestion
        sslContext.ifPresent(() -> {
            LOGGER.debug("Adding SSL handler");
            pipeline.addLast(sslContext.newHandler(ch.alloc()));
        });
```

Reply:
Done.